src/containers/Layout/Footer.tsx

```diff
@@ -18,17 +18,18 @@ const Footer: React.FC<FooterProps> = ({ isLogin }) => {
         <PageHeaderToolsItem>
           <p className="footer-text">An open source clinical decision support UI platform created by&nbsp;
```

Comment:
```suggestion
          <p className="footer-text">An open source neural network platform created by&nbsp;
```
This was the suggestion from the call with FNNDSC